lib/virtualbox.js

```diff
@@ -14,6 +14,9 @@ const execFile = require('child_process').execFile,
 let vBoxManageBinary,
   vbox_version;
  
+function execFile(binary, args, func){
+  exec(binary + args.join(" "), func)
```

Comment:
This still allows for arbitrary command injection. All a user has to do is pass a `; echo "Pwned"` or `| grep "Pwned"` as the first element in the `args` array for it to execute arbitrary code. We should stick to execFile because it doesn't spawn a shell by default.

See the docs here: https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback

Reply:
I agree with that, but unfortunately on my setup it does not work with execFile, as it can not find *any* binary in the path. ExecFile on my Ubuntu only executes full absolute paths for some reason. 
Are you able to integrate (cherry pick) only the clone-related changes or do you want me to move those to a separate branch for merging?

Reply:
Moving to a separate branch would be great, in future we can add an argument to a constructor that will allow folks to customize how they want to execute virtualbox. I'll open an issue and put it on the roadmap.

Reply:
I have opened the revised PR #85